### ISSUE TITLE
How to fully uninstall anvil-runtime?

### ISSUE THREAD

**Issue by jkleckner** [anvil-works/anvil-runtime#41]
After listening to a podcast, I tried creating a virtual env (conda py 3.7) and installed/ran the todo application on my personal Mac (Catalina).

I can't find a description of how to fully uninstall it.  After removing the virtual environment and fully removing the contents of the temporary application directory for the todo application, the route http://127.0.0.1:3030 continues to serve up content including javascript to connect to services like Stripe even after a reboot.  This is quite alarming.

How can I reverse this without having to do a complete reinstall of Catalina or reversion to a recent Time Machine backup?

Update:

Anvil is installing a persistent service worker to support "PWA".  See comment below to clear the data associated with it.
Note that this means any Chrome browser that visits the application will have this running until you do clear it.

**Comment by jkleckner** [comment]
This seems to be some remnant left in Chrome.  Clearing the cache and cookies for that URL or a hard-reload in dev doesn't get rid of it and I don't want to clear all of my stored data.  There isn't anything served on the 3030 port.  Will look into this more tomorrow.

**Comment by jkleckner** [comment]
Ok, fresh in the morning I was able to locate the culprit.

There was a persistent service worker that had been installed that was serving up authentication to Stripe, Google, and a lot of others.

No amount of clearing cookies or app data would solve this - I didn't try nuke of everything since that is very disruptive indeed.

So I ran across the solution [1] which was "How to Force Remove Service Worker from Chrome".

Weirdly hard to find for me, who is not a web developer by day.

[1] https://dev.to/damcclean/how-to-force-remove-service-worker-from-chrome-1n2f
